Readme.md

```diff
@@ -125,9 +125,22 @@ Custom HTML file:
 ```html
 $ echo \
   '<!doctype html>
-  <link rel="stylesheet" href="/style.css" />
-  <script src="/elm.js"></script>
-  ' > index.html
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Elm Live Demo</title>
+    <link rel="stylesheet" href="/style.css" />
+    <script src="/elm.js"></script>
+  </head>
+
+  <body>
+    <div id="root"></div>
+  </body>
+
+  <script>
+    Elm.Main.embed(document.getElementById("root"));
+  </script>
```

Comment:
So if we want to be HTML5-compliant, we’d have to place this script inside the `<body>`. It probably makes more sense to place the `<script src="/elm.js"></script>` inside the body as well, just before this one.

And in lieu of making this as lightweight as possible, how about
* getting the `id` out of the div,
* `document.querySelector('div')` instead of `getElementById`
* and squashing the `<script>\nElm.Main.…` into one line?